### PR TITLE
updated descritpions

### PR DESCRIPTION
**Pull request by Jamie-Atkinson**

### CODE REVIEW

File: data/discovery/field/change-date.yaml [hunk 1]
```diff
@@ -3,4 +3,4 @@ datatype: datetime
 field: change-date
 phase: discovery
 register: ''
-text: Start datetime for the applicability of a change to a field value.
+text: A character (either 1 or n) that explains if a field in a register can contain multiple values.
```
Comment (endamccormack) [inline data/discovery/field/change-date.yaml]: there is an unchanged change-date in alpha

File: data/discovery/field/website.yaml [hunk 2]
```diff
@@ -2,5 +2,5 @@ cardinality: '1'
 datatype: string
 field: website
```
Comment (endamccormack) [inline data/discovery/field/website.yaml]: the datatype in alpha is string

File: data/discovery/field/change-date.yaml (hunk 1 above)
Comment (Jamie-Atkinson) [inline data/discovery/field/change-date.yaml]: thanks

File: data/discovery/field/website.yaml (hunk 2 above)
Comment (Jamie-Atkinson) [inline data/discovery/field/website.yaml]: i was not even looking for that, good catch

File: data/alpha/field/end-date.yaml [hunk 3]
```diff
@@ -3,4 +3,4 @@ datatype: datetime
 field: end-date
 phase: alpha
 register: ''
-text: End datetime for the applicability of a register entry.
+text: The date a record stopped being relevant to a register. For example, the date a school closed down.
```
Comment (endamccormack) [inline data/alpha/field/end-date.yaml]: this in beta is different

File: data/alpha/field/field.yaml [hunk 4]
```diff
@@ -3,4 +3,4 @@ datatype: string
 field: field
 phase: alpha
 register: field
-text: Field name of register entry.
+text: The name of a field. This field could appear in more than one register.
```
Comment (endamccormack) [inline data/alpha/field/field.yaml]: this is SLIGHTLY differently phrased in beta

File: data/alpha/field/local-authority-eng.yaml [hunk 5]
```diff
@@ -1,6 +1,6 @@
-text: "A local authority in England."
+text: The local authority's ISO 3166-1 alpha 2 code. Subsidiary authorities are given a Standard Names and Codes (SNAC) code instead.
```
Comment (endamccormack) [inline data/alpha/field/local-authority-eng.yaml]: this is different in beta

File: data/discovery/field/website.yaml (hunk 2 above)
Comment (Jamie-Atkinson) [inline data/discovery/field/website.yaml]: if i change this one to url if fails the tests

**Comment by endamccormack** [comment]
It would be so much nicer if we cleaned up whitespace and the quotes for consistency, if you dont think thats part of your story then create a card maybe?

File: data/discovery/field/website.yaml (hunk 2 above)
Comment (endamccormack) [inline data/discovery/field/website.yaml]: Well thats a sign of a bigger problem, fix it if you can or leave it and create a card

**Pull request closed without merging**